dds_web/web/user.py

```diff
@@ -28,13 +29,35 @@
 import dds_web.api.errors as ddserr
 from dds_web.api.schemas import user_schemas
 from dds_web import mail
-import flask_mail
+
+
+auth_blueprint = flask.Blueprint("auth_blueprint", __name__)
+
+####################################################################################################
+# ERROR HANDLING ################################################################## ERROR HANDLING #
+####################################################################################################
+
+
+class UserDeletionErrorWeb(werkzeug.exceptions.BadRequest):
+    def __init__(self, message):
+        super().__init__()
+        self.message = message
+
```

Comment:
Bugs me a bit that we can't use the same exceptions for the api and web. Feels like there should be a way of doing this instead of having duplicates?

Reply:
Hopefully fixed now!